rpm/qt5-qpa-hwcomposer-plugin.spec

```diff
@@ -24,6 +24,7 @@ BuildRequires:  pkgconfig(libsync)
 %endif
 BuildRequires:  pkgconfig(hybris-egl-platform)
 BuildRequires:  pkgconfig(android-headers)
+BuildRequires:  pkgconfig(libhwc2)
```

Comment:
I added libhwc2 as part of libhybris package. So we should not need this here and can maybe merge this to master even instead of a special branch, like we discussed?